backend/src/test/java/com/google/models/AlertTest.java

```diff
@@ -0,0 +1,112 @@
+package com.google.models;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import java.util.*;
+import com.google.blackswan.mock.*;
+
+/** Contain tests for methods in {@link Alert} class. */
+@RunWith(JUnit4.class)
+public final class AlertTest {
+  private static final int TIMESTAMP_CONSTANT = 1;
+  private static final AnomalyGenerator dummyAnomalyGenerator = new DummyAnomalyGenerator();
+  private static final LocalServiceTestHelper helper = 
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  private Alert alert;
+
+  @Before
+  public void setUp() throws Exception {
+    helper.setUp();
+    // TODO: Find out whether it's better to set parameters for Alert as private static variables,
+    //       since they're used later to test getters. 
+    alert = new Alert(Timestamp.getDummyTimestamp(TIMESTAMP_CONSTANT), dummyAnomalyGenerator.getAnomalies(), 
+        Alert.UNRESOLVED_MESSAGE);
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void getTimestamp_workingGetter() {
+    assertEquals(alert.getTimestamp(), Timestamp.getDummyTimestamp(TIMESTAMP_CONSTANT));
+  }
+
+  @Test
+  public void getAnomalies_workingGetter() {
+    // Check to see if List<Anomaly> returned is same as List<Anomaly> used to generate alert.
+    assertEquals(alert.getAnomalies(), dummyAnomalyGenerator.getAnomalies());
+  }
+
+  @Test
+  public void getResolvedStatus_workingGetter() {
+    assertEquals(alert.getResolvedStatus(), Alert.UNRESOLVED_MESSAGE);
+  }
+
+  @Test
+  public void setResolvedStatus_workingSetter() {
+    alert.setResolvedStatus(Alert.RESOLVED_MESSAGE);
+
```

Comment:
nit: remove new line

Reply:
the new line is for separation of **act** and **assert** section of a test. is that ok with you?